app/parsers/bulkrax/application_parser.rb

```diff
@@ -358,16 +359,6 @@ def zip
       end
     end
 
-    # Is this a file?
-    def file?
-      parser_fields&.[]('import_file_path') && File.file?(parser_fields['import_file_path'])
-    end
-
-    # Is this a zip file?
-    def zip?
-      parser_fields&.[]('import_file_path') && ::Marcel::MimeType.for(File.new(parser_fields['import_file_path'])).include?('application/zip')
-    end
-
```

Comment:
@jeremyf should we be `raise`ing errors here instead or removing them like how we do in [#records](https://github.com/samvera-labs/bulkrax/blob/main/app/parsers/bulkrax/application_parser.rb#L64), for example?

Reply:
No…we need the method to be on parser, but are relying on the `delegate` declaration to add that method.

In the case of a `NotImplementedError` that is usually used for an `@abstract` class to signify that a descendant should implement that method.